### PR TITLE
Adding default value column for command args, event fields and struct items

### PR DESCRIPTION
**Pull request by dhchandw**
- Default values can be defined for command args, event fields and struct items. These default values can be accessed through zcl static helpers
- Added tests

### CODE REVIEW

**Review by paulr34 — APPROVED** [review]

File: test/gen-matter-1.test.js [hunk 1]
```diff
@@ -293,6 +293,19 @@ test(
...
+    // Testing default values for command arguments, event fields and struct items
+    expect(sdkExt).toContain('ProductID - int16u - default_value=0x01')
+    expect(sdkExt).toContain(
+      'RequestorCanConsent - boolean - default_value=false'
+    )
+    expect(sdkExt).toContain(
+      'Struct name: ChannelInfoStruct, Struct Item Name: MajorNumber, Struct Item Type: int16u, Struct Default Value: 0xFFFF'
+    )
+    let eventOut = genResult.content['events.out']
+    expect(eventOut).toContain(
```
Comment (brdandu) [inline test/gen-matter-1.test.js:306]: Add a not contain test too just to cover negative test cases

File: zcl-builtin/matter/data-model/chip/channel-cluster.xml [hunk 2]
```diff
@@ -54,7 +54,7 @@ limitations under the License.
 
   <struct name="ChannelInfoStruct">
     <cluster code="0x0504"/>
-    <item name="MajorNumber"       type="INT16U"/>
+    <item name="MajorNumber"       type="INT16U" default="0xFFFF"/>
```
Comment (brdandu) [inline zcl-builtin/matter/data-model/chip/channel-cluster.xml:57]: what happens if `default=null` ?

**Review by brdandu — APPROVED** [review]
